src/lib/components/PrismicImage.svelte

```diff
@@ -1,34 +1,63 @@
 <script lang="ts">
-	import * as prismicT from "@prismicio/types";
+	import * as prismicH from "@prismicio/helpers"
+	import type * as prismicT from "@prismicio/types";
+	
+	/**
+	 * A Prismic Image field.
+	 */
 	export let field: prismicT.ImageField;
-	let className: string;
 
-	// creates a `class` property, even
-	// though it is a reserved word
-	export { className as class };
+	/**
+	 * An object of Imgix URL API parameters to transform the image.
+	 *
+	 * See: https://docs.imgix.com/apis/rendering
+	 */
+	export let imgixParams: Parameters<typeof prismicH.asImageSrc>[1] | null | undefined = null;
```

Comment:
Is there a benefit to using `null`? If not, `undefined` is preferred generally speaking.

Reply:
✅